### PR TITLE
Update .config.prowgen for some of the test runs  

### PR DESCRIPTION
**Pull request by aviavraham**

### CODE REVIEW

File: ci-operator/config/maistra/maistra-test-tool/.config.prowgen
Comment (dfrazzette) [inline ci-operator/config/maistra/maistra-test-tool/.config.prowgen:1]: This file has already been created and merged, so not needed.

File: ci-operator/config/konveyor/tackle-ui-tests/.config.prowgen
Comment (dfrazzette) [inline ci-operator/config/konveyor/tackle-ui-tests/.config.prowgen:1]: This file is part of pending [PR#55645](https://github.com/openshift/release/pull/55645) for MTA testing on FIPS and can be removed.

File: ci-operator/config/openshift-pipelines/release-tests/.config.prowgen
Comment (dfrazzette) [inline ci-operator/config/openshift-pipelines/release-tests/.config.prowgen:1]: This `.config.prowgen` file has already been created and merged. This file can be removed.

File: ci-operator/config/maistra/maistra-test-tool/.config.prowgen
Comment (aviavraham) [reply ci-operator/config/maistra/maistra-test-tool/.config.prowgen:1]: @dfrazzette What are your expectations? 
I think this commit  will have no impact on the file if they are identical  
Would you like me to delete it? or should I delete this commit?

**Comment by aviavraham** [comment]
@dfrazzette I removed the requested files from the PR.

**Comment by dfrazzette** [comment]
The `.config.prowgen` file updates have been made separately as part of the OCP 4.18 expansion changes. Closing.

**Pull request closed without merging**